force-app/main/default/flows/Auto_Registration_for_Welcome_Session.flow-meta.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>54.0</apiVersion>
+    <description>Auto Registration for Welcome Session at Dreamforce Events</description>
+    <interviewLabel>Auto Registration for Welcome Session {!$Flow.CurrentDateTime}</interviewLabel>
+    <label>Auto Registration for Welcome Session</label>
+    <processMetadataValues>
+        <name>BuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>FREE_FORM_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>AutoLaunchedFlow</processType>
+    <recordCreates>
+        <description>Attendee who registers for a Dreamforce event will be registered automatically for the Welcome session under the Dreamforce event</description>
+        <name>Register_for_Welcome_Session</name>
+        <label>Register for Welcome Session</label>
+        <locationX>1048</locationX>
+        <locationY>678</locationY>
+        <inputAssignments>
+            <field>Attendee__c</field>
+            <value>
+                <elementReference>$Record.Contact__c</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Session__c</field>
+            <value>
+                <elementReference>Get_Welcome_Session_Id.Id</elementReference>
+            </value>
+        </inputAssignments>
+        <object>Session_Registration__c</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordCreates>
+    <recordLookups>
+        <name>Get_Welcome_Session_Id</name>
+        <label>Get Welcome Session Id</label>
+        <locationX>1048</locationX>
+        <locationY>511</locationY>
+        <assignNullValuesIfNoRecordsFound>false</assignNullValuesIfNoRecordsFound>
+        <connector>
+            <targetReference>Register_for_Welcome_Session</targetReference>
+        </connector>
+        <filterLogic>and</filterLogic>
+        <filters>
+            <field>Name</field>
+            <operator>EqualTo</operator>
+            <value>
+                <stringValue>Welcome Session</stringValue>
```

Comment:
@t-gul the name does not always have to be "Welcome Session". We should not define welcome sessions using their names